src/util/tubemap.js

```diff
@@ -3109,26 +3116,129 @@ function drawNodes(dNodes) {
     .on("mouseover", nodeMouseOver)
     .on("mouseout", nodeMouseOut)
     .on("dblclick", nodeDoubleClick)
+    .on("click", nodeSingleClick)
     .style("fill", config.transparentNodesFlag ? "none" : "#fff")
     .style("fill-opacity", config.showExonsFlag ? "0.4" : "0.6")
     .style("stroke", "black")
     .style("stroke-width", "2px")
     .append("svg:title")
-    .text((d) => getPopUpText(d));
+    .text((d) => getPopUpNodeText(d));
 }
 
-function getPopUpText(node) {
-
-  return (
-    `Node ID: ${node.name}` + (node.switched ? ` (reversed)` : ``) + `\n` +
-    `Node Length: ${node.sequenceLength} bases\n` +
-    `Haplotypes: ${node.degree}\n` +
-    `Aligned Reads: ${
-      node.incomingReads.length +
-      node.internalReads.length +
-      node.outgoingReads.length
-    }`
-  );
+function getPopUpNodeText(node) {
+  return `Node ID: ${node.name}` + (node.switched ? ` (reversed)` : ``) + `\n`;
+}
+
+// Get any node object by name.
+function getNodeByName(nodeName) {
+  if (typeof nodeName !== "string") {
+    throw new Error("Node Names must be strings");
+  }
+  // We just do a scan.
+  // TODO: index!
+  console.log("All nodes:", nodes); //
+  for (let i = 1; i < nodes.length; i++) {
+    // changes i to start from 1 instead of 0
+    if (nodes[i].name === nodeName) {
+      console.log("Found node with name", nodeName, "at index ", i);
+      return nodes[i];
+    }
+  }
+}
+
+function nodeSingleClick() {
+  /* jshint validthis: true */
+  // Get the node name
+  const nodeName = d3.select(this).attr("id");
+  let currentNode = getNodeByName(nodeName);
+  console.log("Node ", nodeName, " is ", currentNode);
+  if (currentNode === undefined) {
+    console.error("Missing node: ", nodeName);
+    return;
+  }
+  let nodeAttributes = [];
+  nodeAttributes.push([
+    "Node ID:",
+    currentNode.name + currentNode.switched ? "(reversed)" : "",
+  ]);
+  nodeAttributes.push(["Node Length:", currentNode.sequenceLength + " bases"]);
+  nodeAttributes.push(["Haplotypes:", currentNode.degree]);
+  nodeAttributes.push([
+    "Aligned Reads:",
+    currentNode.incomingReads.length +
+      currentNode.internalReads.length +
+      currentNode.outgoingReads.length,
+  ]);
+  nodeAttributes.push([
+    "Number of Reads Visiting Node:",
```

Comment:
This counts reads repeatedly if they visit repeatedly, and should have a name that reflects that.

```suggestion
    "Total Visits:",
```